source/gameengine/GameLogic/Joystick/SCA_Joystick.cpp

```diff
@@ -74,95 +72,82 @@ SCA_Joystick::~SCA_Joystick()
 }
 
 SCA_Joystick *SCA_Joystick::m_instance[JOYINDEX_MAX];
-int SCA_Joystick::m_joynum = 0;
-int SCA_Joystick::m_refCount = 0;
+bool SCA_Joystick::m_joystickupdatestatus = false;
```

Comment:
Can you explain the use of this static variable ? As i saw it's used only in `SCA_JoystickManager` to update self joystick list. But this list is exactly the same as `SCA_Joystick::m_instance`. I made a patch which remove `m_joystick` in `SCA_JoystickManager` and remove `m_joystickupdatestatus`. : http://www.pasteall.org/71764/diff. I noticed no problems currently.